bleak/backends/winrt/client.py

```diff
@@ -113,10 +113,14 @@ class BleakClientWinRT(BaseBleakClient):
     a package that enables Python developers to access Windows Runtime APIs directly from Python.
 
     Args:
-        address_or_ble_device (`BLEDevice` or str): The Bluetooth address of the BLE peripheral to connect to or the `BLEDevice` object representing it.
+        address_or_ble_device (``BLEDevice`` or str): The Bluetooth address of the BLE peripheral
+            to connect to or the ``BLEDevice`` object representing it.
 
     Keyword Args:
         timeout (float): Timeout for required ``BleakScanner.find_device_by_address`` call. Defaults to 10.0.
+        use_cached_services (bool): ``True`` allows Windows to fetch the services, characteristics and descriptors
+            from the Windows cache instead of reading them from the device. Can be very much faster for known devices.
```

Comment:
We should probably mention that this is not a good idea for DIY devices where the attribute layout may change frequently.

Reply:
I was planning on having default False and write about the `use_cached_services` as an optional boost in the documentation, for those who are confident about their device.

Do you think it should be the other way around? Use cached unless you know that your device is changing between connects? It would be faster for general users, but really annoy the Ones not only playing around. I don't know what is preferable.

How many of Bleak users also write their own peripheral? 25% at best is my guess.

Reply:
We actually used to have the cached enabled by default, but I removed it because we kept having issues reported that "bleak isn't working". I don't know that it has been long enough to know if that actually solved any problems or not. But I think you have the right idea here that it should not use cache by default.

Reply:
> How many of Bleak users also write their own peripheral? 25% at best is my guess.

I don't have a good guess, but that is a significant percentage. I also don't think everyone is experiencing the long time delays when discovering services like you are.

Reply:
Added text warning about using it with devices created by oneself.